docs/source/model_design/vivarium_model_components/risk_factors/risk_correlation/index.rst

```diff
@@ -1,1213 +1,152 @@
-exposures, after scaling and shifting so that the TMREL is 0.0 and a one
-unit increment gives the relative increase in disease burden reported by
-GBD.
+For example, a simulant might have their propensities for height and weight sampled at birth from a
+multivariate uniform distribution with correlation.
+If the simulant receives a propensity of 0.85 (85th percentile) in height and 0.80 (80th percentile) in weight,
+they are initially assigned the height and weight values at those percentiles among newborns.
+But when they age into the next age group, they are assigned the values at those same percentiles in their new
+age group, and so on throughout their life.
+In this way, their height and weight can change over time, but the correlation introduced in the initial sampling
+of propensities remains.
 
-With the log-linear assumption, we can calculate the PAF from first
-principles as a double integral (below :math:`p_{e_1,e_2}` denotes the
-joint probability density of exposure level :math:`(e_1,e_2)` in the
-population:
+.. todo::
+  Be more exact about the properties of the propensity correlation approach.
+  I believe that it only exactly maintains the correlation coefficient with some distributional assumptions.
 
-..
-  :raw-latex:`\begin{align*}
-  i_{\text{GBD}} &= \int_{e_1} \int_{e_2} i_{e_1, e_2} p_{e_1,e_2} de_1 de_2\\
-  &= \int_{e_1} \int_{e_2} i_0 \cdot \left(\text{RR}_1\right)^{e_1}\cdot \left(\text{RR}_2\right)^{e_2}
-  p_{e_1,e_2} de_1 de_2\\
-  &= \int_{e_1} \int_{e_2} \left(1-\text{PAF}\right) \cdot i_{\text{GBD}} \cdot \left(\text{RR}_1\right)^{e_1}\cdot \left(\text{RR}_2\right)^{e_2}
-  p_{e_1,e_2} de_1 de_2\\
-  \end{align*}`
+Risks with effects on the same outcome
+--------------------------------------
 
-.. math::
-  :nowrap:
+When we introduce correlation between two risk factors that have effects on the same outcome, we have to
+be especially careful about double-counting issues.
 
-  \begin{align*}
-  i_{\text{GBD}} &= \int_{e_1} \int_{e_2} i_{e_1, e_2} p_{e_1,e_2} de_1 de_2\\
-  &= \int_{e_1} \int_{e_2} i_0 \cdot \left(\text{RR}_1\right)^{e_1}\cdot \left(\text{RR}_2\right)^{e_2}
-  p_{e_1,e_2} de_1 de_2\\
-  &= \int_{e_1} \int_{e_2} \left(1-\text{PAF}\right) \cdot i_{\text{GBD}} \cdot \left(\text{RR}_1\right)^{e_1}\cdot \left(\text{RR}_2\right)^{e_2}
-  p_{e_1,e_2} de_1 de_2
-  \end{align*}
+The population-attributable fractions (PAFs) must be calculated correctly,
+or the population-level incidence of the outcome will
+not validate to GBD.
+**The usual Vivarium strategy to calculate joint PAFs (PAFs of more than one risk factor on an outcome)
+is biased when the risk factors are correlated.**
 
-Dividing through and cancelling terms yields a solution:
+The true population attributable fraction for the combined effect of two risk factors,
+:math:`r1` and :math:`r2`, on an outcome :math:`O` such as a mortality rate is (assuming no effect modification, as we typically do):
```

Comment:
Oh, are you saying "no effect modification" is equivalent to the assumption that

RR(r1, r2) = RR(r1)*RR(r2)

I hadn't thought about it that way before. Do you mean that r1 is not an effect modifier for r2, and vice versa?

Reply:
Yes -- in my brain that is the definition of "no effect modification" though I admit I am not too familiar with how people usually use the term.

Reply:
Which is part of why I have always been a bit confused by "X modifies the effect of Y" constructions -- it should always be reciprocal, I think.

Reply:
Yeah, another way to say it would be there is a statistical interaction between the effect of R1 and R2 on O. While they are the synonymous, I think effect modification is used more often when there's a focus on a specific exposure/outcome pair and interaction is used more often in other contexts. We've tended to use the term effect modification, but interaction might be more intuitive because were rarely focusing on a single exposure/outcome like you would in some epi investigation of causality.